dev/doc/MERGING.md

```diff
@@ -30,8 +30,8 @@ When maintainers receive a review request, they are expected to:
 * If they are the assignee, check if all reviewers approved the PR. If not,
   regularly ping the author (if changes should be implemented) or the reviewers
   (if reviews are missing). The assignee ensures that any requests for more
-  discussion have been granted. When the discussion has converged and all
-  reviewers have approved the PR, the assignee is expected to follow the merging
+  discussion have been granted. When the discussion has converged and ALL
+  REVIEWERS(*) have approved the PR, the assignee is expected to follow the merging
```

Comment:
All code owners might be more clear.

Reply:
I think it is really all reviewers, as code owners may have delegated the review.

Reply:
OK